app/src/behavior.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <sys/util.h>
+#include <logging/log.h>
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+#include <zmk/matrix.h>
+#include <zmk/sensors.h>
+#include <zmk/keymap.h>
+#include <drivers/behavior.h>
+#include <zmk/behavior.h>
+
+#include <zmk/event_manager.h>
+#include <zmk/events/position_state_changed.h>
+#include <zmk/events/behavior_state_changed.h>
+#include <zmk/events/sensor_event.h>
+
+int zmk_behavior_state_changed_listener(const zmk_event_t *zmk_ev) {
+    struct zmk_behavior_state_changed *ev = as_zmk_behavior_state_changed(zmk_ev);
+    if (ev == NULL) {
+        return 0;
```

Comment:
If we're getting any other event type, something is really amiss, and I feel like we should be returning an error (we might not be doing this elsewhere, but we should), I would suggest `-ENOTSUP` or somesuch.

Reply:
Bump.